data/json/monstergroups/bugs.json

```
@@ -10,6 +10,21 @@
       { "monster": "mon_ant_queen", "freq": 0, "cost_multiplier": 0 }
     ]
   },
+  {
+    "type": "monstergroup",
+    "name": "GROUP_ANT_SURFACE",
+    "default": "mon_ant_small",
+    "is_animal": true,
+    "monsters": [
+      { "monster": "mon_ant_larva", "freq": 40, "cost_multiplier": 0 },
+      { "monster": "mon_ant_soldier", "freq": 90, "cost_multiplier": 5 },
+      { "monster": "mon_ant_queen", "freq": 0, "cost_multiplier": 0 },
```

Neither ant larvae nor ant queens make sense above ground, might as well remove them while you're here.
```suggestion
      { "monster": "mon_ant_soldier", "freq": 90, "cost_multiplier": 5 },
```